contrib/podmanimage/README.md

```diff
@@ -16,11 +16,12 @@ default to `/`.
 
 The container images are:
 
-  * quay.io/containers/podman - This image is built using the latest stable version of Podman in a Fedora based container.  Built with [podmanimage/stable/Dockerfile](stable/Dockerfile).
-  * quay.io/podman/stable - This image is built using the latest stable version of Podman in a Fedora based container.  Built with [podmanimage/stable/Dockerfile](stable/Dockerfile).
-  * quay.io/podman/upstream - This image is built using the latest code found in this GitHub repository.  When someone creates a commit and pushes it, the image is created.  Due to that the image changes frequently and is not guaranteed to be stable.  Built with [podmanimage/upstream/Dockerfile](upstream/Dockerfile).
-  * quay.io/podman/testing - This image is built using the latest version of Podman that is or was in updates testing for Fedora.  At times this may be the same as the stable image.  This container image will primarily be used by the development teams for verification testing when a new package is created.  Built with [podmanimage/testing/Dockerfile](testing/Dockerfile).
-  * quay.io/podman/stable:version - This image is built manually using a Fedora based container.  An RPM is first pulled from the [Fedora Updates System](https://bodhi.fedoraproject.org/) and the image is built from there.  For more details, see the Containerfile used to build it, [podmanimage/stable/manual/Containerfile](stable/manual/Containerfile).
+  * `quay.io/containers/podman` - This image is built using the latest stable version of Podman in a Fedora based container.  Built with [podmanimage/stable/Dockerfile](stable/Dockerfile).
+  * `quay.io/podman/stable` - This image is built using the latest stable version of Podman in a Fedora based container.  Built with [podmanimage/stable/Dockerfile](stable/Dockerfile).
+  * `quay.io/podman/stable:<version>` - This image is built and pushed daily, when a new podman version becomes available in Fedora.  An RPM is first pulled from the [Fedora Updates System](https://bodhi.fedoraproject.org/) and the image is built from there.  For more details, see the Containerfile used to build it, [podmanimage/stable/manual/Containerfile](stable/manual/Containerfile).
```

Comment:
Just double checking, just `quay.io/containers/podman:latest` and `quay.io/podman/stable:latest` are pushed daily.  The `quay.io/podman/stable:<version>` should only be pushed if not present on the repo....

Reply:
>  just quay.io/containers/podman:latest and quay.io/podman/stable:latest are pushed daily

Actually, I've been struggling with this.  We only push a tag 'master' and not 'latest'.  I can update the docs, and/or the code to also push 'latest' if you like.

> The quay.io/podman/stable:<version> should only be pushed if not present on the repo.

Correct, [that's this bit of code here.](https://github.com/containers/podman/pull/10150/files#diff-691cbdaf78b3bc338d4c6458c8d7eccae7f3a89afbc1ed27674bbf1fe53aa33bR75-R83).  The build happens daily, but we only push if the podman version (as reported by podman) doesn't have a tag.

Reply:
@cevich, I'd not push a master tag, if we can, I'd let that whither on the vine.  Let's just push the `:latest` tag.
I'll go suggest a change to the `:<version>` doc here, it's not currently clear when that gets pushed.

Reply:
Okay, fixed both.  PTAL when you get a chance.